substrate-node/pallets/test_offchain_worker.py

```diff
@@ -0,0 +1,99 @@
+from random import randbytes
```

Comment:
Curiosity
How this python file is used for testing?

Reply:
It's more a temporary testing file. This will be reworked once we implement integration tests later. This script simplifies creating a contract (the steps you need to take to submit the contract) and allows you to use it to test the billing when you have more then one node. You can run the script after setting up your network as described here: https://docs.substrate.io/tutorials/get-started/simulate-network/.

Reply:
It would be good to have an automated way of executing those kinds of tests. These scripts contain the first steps towards that.